src/common/txsplain.js

```diff
@@ -607,6 +637,10 @@ angular.module('txsplain', [])
           d += renderPayment()
         } else if (tx.tx.TransactionType === 'TrustSet') {
           d += renderTrustSet()
+        } else if (tx.tx.TransactionType === 'EscrowCreate') {
+          d += renderEscrowCreate()
+        } else if (tx.tx.TransactionType === 'EscrowFinish') {
+          d += renderEscrowFinish()
```

Comment:
There is also a EscrowCancel tx type that needs to be rendered.  Text something like "100 XRP was returned to rXYZ..."